test/test_readCSV.py

```diff
@@ -1,29 +1,31 @@
 import unittest
 import csv
+from src import read_csv
 
-#TODO anyone know how to do this better than what it is now?
-import sys, os
-sys.path.insert(0, os.path.abspath('..'))
-from src import readCSV
 
 class TestReadCSV(unittest.TestCase):
+    """Class for testing read_csv.py"""
 
-    def test_transfromCSVFromBadFormatToBetterFormat(self):
-        fileName = "TestCSV.csv"
-        betterFileName = "better" + fileName
+    def test_transform_csv_from_bad_format_to_better_format(self):
+        csv_path = "../data/csv_files/"
```

Comment:
csv_path = os.path.dirname(__file__) + "/../data/csv_files/"

The unittest discovery runs from a different directory, so we need to grab the directory of this file in order to navigate appropriately from both the unittest command and just running test files individually on our own machines